### ISSUE TITLE
Packet profiling hook isn't added when using Thermos

### ISSUE THREAD

**Issue by LunNova** [MinimallyCorrect/TickProfiler#40]
report from Rubbertjuh:

```
[01:27:07] [Netty IO #1/WARN] [JavaPatcher/]: Didn't find any method calls to replace in net.minecraft.util.MessageSerializer.encode(io.netty.channel.ChannelHandlerContext,java.lang.Object,io.netty.buffer.ByteBuf). Class: null, method: func_148840_b, index: -1
[01:27:07] [Netty IO #1/WARN] [JavaPatcher/]: 
java.lang.RuntimeException: Method not found: scheduleOutboundPacket() was not found in net.minecraft.util.MessageSerializer
    at me.nallar.javapatcher.mappings.MethodDescription.inClass(MethodDescription.java:173) ~[MethodDescription.class:?]
    at me.nallar.javapatcher.patcher.Patcher$PatchMethodDescriptor.run(Patcher.java:369) [Patcher$PatchMethodDescriptor.class:?]
    at me.nallar.javapatcher.patcher.Patcher$ClassPatchDescriptor.runPatches(Patcher.java:501) [Patcher$ClassPatchDescriptor.class:?]
    at me.nallar.javapatcher.patcher.Patcher.patch(Patcher.java:191) [Patcher.class:?]
    at me.nallar.modpatcher.ModPatcher.postSrgTransformationHook(ModPatcher.java:82) [ModPatcher.class:?]
    at me.nallar.modpatcher.ModPatcher.transform(ModPatcher.java:121) [ModPatcher.class:?]
    at net.minecraft.launchwrapper.LaunchClassLoader.runTransformers(LaunchClassLoader.java:279) [launchwrapper-1.12.jar:?]
    at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:176) [launchwrapper-1.12.jar:?]
    at java.lang.ClassLoader.loadClass(ClassLoader.java:424) [?:1.8.0_66]
    at java.lang.ClassLoader.loadClass(ClassLoader.java:357) [?:1.8.0_66]
    at net.minecraft.network.NetworkSystem$1.initChannel(NetworkSystem.java:89) [nd.class:?]
    at io.netty.channel.ChannelInitializer.channelRegistered(ChannelInitializer.java:70) [ChannelInitializer.class:?]
    at io.netty.channel.DefaultChannelHandlerContext.invokeChannelRegistered(DefaultChannelHandlerContext.java:162) [DefaultChannelHandlerContext.class:?]
    at io.netty.channel.DefaultChannelHandlerContext.fireChannelRegistered(DefaultChannelHandlerContext.java:148) [DefaultChannelHandlerContext.class:?]
    at io.netty.channel.ChannelInitializer.channelRegistered(ChannelInitializer.java:73) [ChannelInitializer.class:?]
    at io.netty.channel.DefaultChannelHandlerContext.invokeChannelRegistered(DefaultChannelHandlerContext.java:162) [DefaultChannelHandlerContext.class:?]
    at io.netty.channel.DefaultChannelHandlerContext.fireChannelRegistered(DefaultChannelHandlerContext.java:148) [DefaultChannelHandlerContext.class:?]
    at io.netty.channel.DefaultChannelPipeline.fireChannelRegistered(DefaultChannelPipeline.java:730) [DefaultChannelPipeline.class:?]
    at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:441) [AbstractChannel$AbstractUnsafe.class:?]
    at io.netty.channel.AbstractChannel$AbstractUnsafe.access$100(AbstractChannel.java:373) [AbstractChannel$AbstractUnsafe.class:?]
    at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:417) [AbstractChannel$AbstractUnsafe$1.class:?]
    at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:354) [SingleThreadEventExecutor.class:?]
    at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:348) [NioEventLoop.class:?]
    at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:101) [SingleThreadEventExecutor$2.class:?]
    at java.lang.Thread.run(Thread.java:745) [?:1.8.0_66]
```

**Comment by LunNova** [comment]
Will cause `/profile p` to give no results.

**Comment by origamerpro** [comment]
http://pastebin.com/HQ08hmaR - Thermos 58 1664

**Comment by LunNova** [comment]
No longer planning to fix this, not supporting 1.7.10 at this point.
